sktime_dl/deeplearning/fcn/_classifier.py

```diff
@@ -83,6 +78,9 @@ def build_model(self, input_shape, nb_classes, **kwargs):
                       metrics=['accuracy'])
 
         # if user hasn't provided a custom ReduceLROnPlateau via init already, add the default from literature
+        if self.callbacks is None:
```

Comment:
Perhaps we should also introduce a `check_callbacks` utility function for input validation, I'm guessing this is something we do in every estimator

Reply:
Personally I prefer to leave it, since it's just two lines of code. This way the reader can see instantly what it does.

Reply:
My thinking was that since we're setting a default, it'll be easier in the future if we want to modify/replace that, but happy to leave it as it is